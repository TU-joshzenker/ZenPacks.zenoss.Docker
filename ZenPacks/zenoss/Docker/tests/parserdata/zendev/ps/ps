b8a2160dd590a1306a97dce2a59a881a8fccca6691250f46aee1ad8e57945a0d|Up 24 hours|10.43 MB (virtual 511.5 MB)
5f5afa6b219d4d7d65e1434dddcba97bb2798208e317521619d418914beb956f|Up 24 hours|10.42 MB (virtual 511.5 MB)
648b8f9065c31f38ae32215fc9eb50074ad13096f1d41020e6fcf3c235d3a44a|Up 24 hours|10.42 MB (virtual 511.5 MB)
e6a04a3e3218a61abb7b1c6f3b198e4bdcdfd77692f20d72a2f72e656fb8291d|Up 24 hours|2.226 MB (virtual 503.2 MB)
1572c9ef172215129a19dab9aeb72524818e8753b387d8246889c99dc7c58571|Up 24 hours|2.23 MB (virtual 503.3 MB)
80a886ffab85bb6b9e0cb604109c1c04c67ba887f35a5a38e31187cde1796c21|Up 24 hours|2.383 MB (virtual 503.4 MB)
6f9454b508460f466eb2717215297f21dd3a5db45ddfa6203e15579d887f7c50|Up 24 hours|2.195 MB (virtual 2.257 GB)
47342e3152250664c89bd9ab9b3d702a29c110c0414b257245835898b136c05c|Up 24 hours|2.195 MB (virtual 2.257 GB)
9d720025b9e4027f7e7eea7d9b8cba646bd6f3bc020e1a04141018c21408f106|Up 24 hours|2.162 MB (virtual 2.257 GB)
69b3cbe870065c65b7fcff230723ad6f502435ef284401527725659d4ffeff16|Up 24 hours|2.162 MB (virtual 2.257 GB)
9935750d2a407a25eab19fb20a17e84f291a722641b6086c654a6f6ad68c3676|Up 24 hours|1.775 MB (virtual 2.257 GB)
f2f34fbad4284b4f7da9f299772547284518d9e5e2beec2a7e5cd0c595eab12d|Up 24 hours|2.162 MB (virtual 2.257 GB)
68a24a3e06734753724a532fe63a247f2faf2cef0d8b353e36cd38c751a6d820|Up 24 hours|1.895 MB (virtual 2.257 GB)
d84aa32fabc47e0b2cad3dbe629a820ffcab5ba8b47a977da0a98acfea6ef73e|Up 24 hours|11.07 MB (virtual 512.1 MB)
1d73de2b8c1c2428933cae84caea58135d07131c70238e0f0d4cc0ea00da4a63|Up 24 hours|2.195 MB (virtual 2.257 GB)
8d42d85b741acb35cd22ed66f9daa856da2f9623ccb28251128621b96da2fd69|Up 24 hours|29.78 MB (virtual 530.8 MB)
c686c5cb3cb57362f300d7ef573a0037deda07392e47717bf6f9c24da20d9947|Up 24 hours|2.162 MB (virtual 2.257 GB)
fe715ab2b307a141c5407deec5b5145b9d2d04f01133361e48d3353d5af1295f|Up 24 hours|2.567 MB (virtual 2.257 GB)
e722b2a070a58d62b0faec3d34aba61a5055ae0a97827806b16d92973b6e6f56|Up 24 hours|1.799 MB (virtual 2.257 GB)
8327ee7a6fdda62c3a031964c9c9e8741070c2864200226e2d59fbcdca6ad552|Up 24 hours|1.895 MB (virtual 2.257 GB)
944f68c15f82f1364b733c631269827869fcc58286850f614626d81ed1e2eb13|Up 24 hours|2.162 MB (virtual 2.257 GB)
610ea7e9efa3d747c3570e2fdc9e7a326dcf787d9c20f03c7ef41e4f12eb6640|Up 24 hours|2.162 MB (virtual 2.257 GB)
161abb74cc6a046d8e8669311d0ec2538c4361ec47c701651491a4d3bbb7bb67|Up 24 hours|2.162 MB (virtual 2.257 GB)
d6338f5859c7373e326bb08fac736d047fee7ae7799f9c63201d4f04dd9c5c7c|Up 24 hours|2.17 MB (virtual 2.257 GB)
c779b2e7c7a55f629cf91b8bcef2d321f1b7e5a1bdd5055c4abb060ea8205c32|Up 24 hours|2.066 MB (virtual 2.257 GB)
a11798ce3f7dc86205e071c77894ba49b7cf9f9e1329971bcf96a4d221434d7e|Up 24 hours|1.776 MB (virtual 2.257 GB)
26ea56054aa536363b4b74aa9789693d6f41216914faf80e43577075411b7f2b|Up 24 hours|1.771 MB (virtual 2.257 GB)
c437cc74b4a9900b3eeac6809cb77e439b62fd6187c4fd6760348ead55539613|Up 24 hours|27.45 MB (virtual 528.5 MB)
b9f6e235a700b809c603600df000115c0e44ae76ac0d6e4bfa2142d4d9a46937|Up 24 hours|2.458 MB (virtual 2.257 GB)
e42e928800aa087cf83841a0c4230c054a04a3fc97425467ce7688fddd248d4f|Up 24 hours|123.2 kB (virtual 780 MB)
cf42c9cb23112d232aa4e0be0002ef4783b583b296f6fd6aa9fa50813cda5553|Up 24 hours|66.33 kB (virtual 416.7 MB)
9f4c78631edcce0ca072e3771b50924314acd950e734575ec61ffd55b5e45b35|Up 24 hours|653.5 kB (virtual 780.5 MB)
81cb5bd275bc9f6b580d109e9ca73176488b92556d1e4f652dc6ab68f6a59c01|Up 24 hours|0 B (virtual 779.9 MB)
f60eada6706a362f66997e0681a09712a9ff7bc428d93b693561c42e93a7e61f|Up 24 hours|1.23 MB (virtual 781.1 MB)
da7ffd3e1394c320a33a41ab33a6d873ba9e316e03fa5a380fa0cd07838e745b|Up 24 hours|34.23 kB (virtual 779.9 MB)
a595a3b74e9457738c6ba691d466bf6319f17792a511421388266d8210e59c73|Up 24 hours|21.37 MB (virtual 801.2 MB)
