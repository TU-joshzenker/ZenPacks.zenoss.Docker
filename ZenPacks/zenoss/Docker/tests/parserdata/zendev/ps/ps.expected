# vim: ft=python
{
    "events": [{
        "device": "zendev",
        "component": "5f5afa6b219d4d7d65e1434dddcba97bb2798208e317521619d418914beb956f",
        "summary": "container status: up 24 hours",
        "severity": 0,
        "eventClassKey": "dockerContainerStatus",
        "eventKey": "dockerContainerStatus",
    }],
    "values": {
        "5f5afa6b219d4d7d65e1434dddcba97bb2798208e317521619d418914beb956f": {
            "size": 10926161.92,
            "size_virtual": 536346624.0,
        },
    },
}
